src/routes/api/index.js

```diff
@@ -1,31 +1,42 @@
-import express from 'express';
-import passport from 'passport';
-
-const usersRouter = express.Router();
-
-usersRouter.post('/users/login', (req, res, next) => {
-  if (!req.body.user.email) {
-    return res.status(422).json({ errors: { email: "can't be blank" } });
-  }
-
-  if (!req.body.user.password) {
-    return res.status(422).json({ errors: { password: "can't be blank" } });
-  }
-  passport.authenticate('local', { session: false }, (
-    err,
-    user,
-    info
-  ) => {
-    if (err) {
-      return next(err);
-    }
-
-    if (user) {
-      return res.json({ user: user.toAuthJSON() });
-    }
-    return res.status(422).json(info);
-  })(req, res, next);
-});
+// import express from 'express';
+// import passport from 'passport';
+
+// const usersRouter = express.Router();
+
+// usersRouter.post('/users/login', (req, res, next) => {
+//   if (!req.body.user.email) {
+//     return res.status(422).json({ errors: { email: "can't be blank" } });
+//   }
+
+//   if (!req.body.user.password) {
+//     return res.status(422).json({ errors: { password: "can't be blank" } });
+//   }
+//   passport.authenticate('local', { session: false }, (
+//     err,
+//     user,
+//     info
+//   ) => {
+//     if (err) {
+//       return next(err);
+//     }
+
+//     if (user) {
+//       return res.json({ user: user.toAuthJSON() });
+//     }
+//     return res.status(422).json(info);
+//   })(req, res, next);
+// });
+
+
+// export default usersRouter;
```

Comment:
I see you commented these lines out. If they are not useful, you should take them out?. I noticed @chibytez and @okafor-chidimma also approved this PR. Do you think the commented lines should be left in the codebase?

Reply:
it will be taken off... I think this code is redundant